components/pickerbutton/stories/template.js

```diff
@@ -31,25 +28,29 @@ export const Template = ({
 	...globals
 }) => {
 	const [_, updateArgs] = useArgs();
+	const { express } = globals;
+
+	try {
+		if (!express) import(/* webpackPrefetch: true */ "../themes/spectrum.css");
+		else import(/* webpackPrefetch: true */ "../themes/express.css");
+	} catch (e) {
+		console.warn(e);
+	}
 
 	return html`
 		<button
 			class=${classMap({
 				[rootClass]: true,
 				[`${rootClass}--textuiicon`]: label && iconType === "ui",
 				[`${rootClass}--uiicononly`]: !label && iconType === "ui",
-				[`${rootClass}--texticon`]: label && iconType !== "ui",
 				[`${rootClass}--icononly`]: !label && iconType !== "ui",
 				[`${rootClass}--${position}`]: typeof position !== "undefined",
 				[`${rootClass}--rounded`]: isRounded,
 				[`${rootClass}--size${size?.toUpperCase()}`]:
 					typeof size !== "undefined",
 				"is-disabled": isDisabled,
 				"is-focused": isFocused,
-				"is-keyboardFocused": isKeyboardFocused,
```

Comment:
saw that `isKeyboardFocused` is still in the controls but we removed the styling here.

Reply:
Yeah the Figma file doesn't have any distinction between focused and keyboard focused like some other ones have so I left the control in so we could still test it but the styling should be the same as focused